modules/web-console/frontend/app/primitives/modal/index.scss

```diff
@@ -55,6 +55,10 @@
     width: 650px;
 }
 
+.modal .modal-stacktrace {
```

Comment:
Placing particular dialog styles into generic modal style breaks isolation of concerns. I'd move this style to  queries module. Optimally, you'd implement trace viewer as component and this style would belong to it, but I don't insist on this.

Reply:
Stack trace dialog moved to separated component.